AdyenUI/ColorScheme.swift

```diff
@@ -0,0 +1,110 @@
+//
+// Copyright (c) 2025 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import UIKit
+
+public struct ColorScheme: Equatable {
+
+    public var background: UIColor
+    public var container: UIColor
+    public var primary: UIColor
+    public var textOnPrimary: UIColor
+    public var action: UIColor
+    public var destructive: UIColor
+    public var textOnDestructive: UIColor
+    public var disabled: UIColor
+    public var textOnDisabled: UIColor
+    public var outline: UIColor
+    public var text: UIColor
```

Comment:
I see some ambiguity here that might lead to additional questions from merchants.
What's the difference between `primary` and `text`? If `primary` is not used for text color, then what is it used for?
And we have `textSecondary` but we have no `secondary`...